typescript/test_application/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "test-application",
+  "version": "0.0.1",
+  "description": "Test application for generated client library.",
+  "engines": {
+    "node": ">=8.13.0"
+  },
+  "scripts": {
+    "test": "c8 --reporter=lcov bash showcase-setup.sh",
+    "codecov": "c8 --reporter=lcov mocha build/test && c8 report",
+    "lint": "gts check",
+    "clean": "gts clean",
+    "fix": "gts fix"
+  },
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/command-line-args": "^5.0.0",
+    "@types/get-stdin": "^5.0.1",
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^11.13.20",
+    "@types/nunjucks": "^3.1.0",
+    "@types/rimraf": "^2.0.2",
+    "assert-rejects": "^1.0.0",
+    "c8": "^5.0.4",
+    "codecov": "^3.0.4",
+    "espower-typescript": "^9.0.0",
+    "google-gax": "^1.6.0",
+    "gts": "^1.0.0",
+    "intelli-espower-loader": "^1.0.1",
+    "mocha": "^6.0.0",
+    "power-assert": "^1.6.0",
+    "rimraf": "^3.0.0",
+    "typescript": "~3.6.0"
+  },
+  "dependencies": {
+    "command-line-args": "^5.0.2",
```

Comment:
I'm kind of sure the test application does not need any of these dependencies. It will be cleaner if it depends on `"showcase": "file:///../../path/to/showcase/client"`, then you will be able to do 

```js
const showcase = require('showcase');
```

and it will use the client library exactly as the real users will do it.